playbooks/roles/selinux/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: 'Ensure libselinux-python is installed'
+  yum:
+    name: 'libselinux-python'
+    state: installed
```

Comment:
This would be a good role to also had enforce/permissive setting of SELinux which is sometimes needed for testing or devel setups.